packages/mongo-schema/schema_validate.js

```
@@ -0,0 +1,284 @@
+  }
+
+  // $push/$addToSet — validate the pushed item against the array item schema
+  for (const op of ['$push', '$addToSet']) {
+    if (!modifier[op]) continue;
+    for (const [key, value] of Object.entries(modifier[op])) {
+      const itemKey = `${key}.$`;
+      const itemDesc = ir.get(itemKey);
+      if (!itemDesc) continue;
+      const pushValue = value && value.$each ? null : value;
+      if (pushValue !== null && pushValue !== undefined) {
+        validateField(ir, itemKey, itemDesc, modifier[op], pushValue, errors, options);
```

ESLint

[error] Replace `ir,·itemKey,·itemDesc,·modifier[op],·pushValue,·errors,·options` with `⏎··········ir,⏎··········itemKey,⏎··········itemDesc,⏎··········modifier[op],⏎··········pushValue,⏎··········errors,⏎··········options,⏎········`

(prettier/prettier)